hamilton/function_modifiers/base.py

```diff
@@ -228,6 +228,63 @@ def transform_dag(
         pass
 
 
+class NodeInjector(SubDAGModifier, abc.ABC):
+    """Injects a value as a source node in the DAG. This is a special case of the SubDAGModifier,
+    which gets all the upstream (required) nodes from the subdag and gives the decorator a chance
+    to inject values into them."""
+
+    @staticmethod
+    def find_injectable_params(nodes: Collection[node.Node]) -> Dict[str, Type[Type]]:
+        """Identifies parameters of this DAG that aren't satisfied by the nodes inside it.
+        These are "injectable", meaning that we can add parameters that feed into them.
```

Comment:
is injectable at all related to the "inject" keyword argument? Otherwise what does parameters refer to here?

Reply:
Yes and no. Its in the superclass, but its the same thing. Parameters are any nodes that this DAG consumes but does not produce.

Reply:
Is it fair to say we're using "parameters" here instead of "required nodes"? It'd be great to standardize on language.

Reply:
Sure? although that's confusing with "parameters" potentially meaning fn parameters. 

These are all internal APIs and quite likely to change...

Reply:
> Sure? although that's confusing with "parameters" potentially meaning fn parameters.

Right, my point exactly. We need to standardize on language.

Reply:
Ahh yeah, just saw that you were referring to the wording I used :)